src/base/_defaults.scss

```diff
@@ -151,3 +151,28 @@ table {
   text-align: left;
   width: 100%;
 }
+
+/*
+ * 1. Removes default margin
+ * 2. Center content
+ * 3. Space between elements inside <figure>
+ */
+
+figure {
+  margin: 0; /* 1 */
+  text-align: center;
+
+  & > * {
+    margin-left: auto; /* 2 */
+    margin-right: auto; /* 2 */
+  }
+
+  & > :not(:first-child) {
```

Comment:
A shorter way of writing this would be `& > * + *`. This also has _slightly_ lower specificity. (You can compare [at this website](https://specificity.keegan.st/).)